simtools/simulator.py

```diff
@@ -208,6 +213,12 @@ def _load_corsika_config_and_model(self, config_data):
             "corsika_parameters_file", None
         )
 
+        # Remove sim_telarray parameters from the CORSIKA config dictionary
```

Comment:
is this necessary to run the code or is it simply that you don't want to have sim_telarray configuration parameter in the corsika config.

Reply:
Yes it's actually necessary cause otherwise the corsika config complains about unknown parameters provided. I would like to find a better solution in the future but as a first draft it's not too bad.

Reply:
OK!